src/index.js

```diff
@@ -10,7 +10,7 @@ import AppContainer from './components/App/container';
 import Trainer from './components/Trainer/Trainer';
 import WebcamCaptureContainer from './components/WebcamCapture/WebcamCaptureContainer';
 import Disclaimer from './components/Disclaimer/Disclaimer';
-import ConfirmationBox from './components/ConfirmationBox/ConfirmationBox';
+import ConfirmationBox from './components/ConfirmationBox/container';
```

Comment:
I don't know if you have been doing this throughout so if you have it's not a big deal, but ```container``` is a fairly generic final file location. I would prefer ```ConfirmationBoxContainer``` etc.